docs/LOCAL_DEPLOYMENT.md

```diff
@@ -78,14 +78,33 @@ If you want to develop and run the batch processing functions container locally,
 
 First, install [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Cportal%2Cv2%2Cbash&pivots=programming-language-python).
 
-
 ```shell
 cd code\batch
 func start
 ```
 
 Or use the [Azure Functions VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions).
 
+#### Debugging the batch processing functions locally
+Add a new file `local.settings.json` in the `batch` folder with content as below and update the `AzureWebJobsStorage` value with the storage account connection string:
+``` json
+{
+    "IsEncrypted": false,
+    "Values": {
+      "FUNCTIONS_WORKER_RUNTIME": "python",
+      "AzureWebJobsStorage": "",
+      "MyBindingConnection": "",
+      "AzureWebJobs.HttpExample.Disabled": "true"
+    },
+    "Host": {
+      "LocalHttpPort": 7071,
+      "CORS": "*",
+      "CORSCredentials": false
+    }
+  }
+```
+Execute the above shell command to run the function locally. You may need to stop the deployed function on the portal so that all requests are debugged locally.
```

Comment:
Should we make this clearer that it's the command fromt he previous section? Maybe add a link to it in the `above shell command` text.